pro/src/utils/csvConverter.ts

```diff
@@ -1,7 +1,12 @@
 import parse from 'csv-parse/lib/sync'
 const CSV_SEMI_COLON_SEPARATOR = ';'
 
-const convertFromCsvToObject = (csv: string) => {
+interface ObjectFromCsv {
```

Comment:
préfixé par i pour interface, `IObjectFromCsv`

Reply:
`because it's unnecessary, adds little value`
C'est leurs opinion.

Ne pas avoir de conflict de nommage entre une interface  `IOfferCategory` et un composant `OfferCategory` je trouve que c'est pratique.